plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -6,12 +6,11 @@ class Release < ActiveRecord::Base
 
     belongs_to :user
     belongs_to :build
+    belongs_to :project
```

Comment:
it blows up when it's not set, so makes sure it is always passed in -> not
necessary to prefill it ? / no other model does that
no other model has project integrity validation -> not necessary here ?

On Fri, Apr 29, 2016 at 4:37 PM, Jon Moter <notifications@github.com> wrote:

> In plugins/kubernetes/app/models/kubernetes/release.rb
> <https://github.com/zendesk/samson/pull/912#discussion_r61658036>:
>
> > @@ -6,12 +6,11 @@ class Release < ActiveRecord::Base
> >
> >      belongs_to :user
> >      belongs_to :build
> > +    belongs_to :project
>
> and maybe a before_validation filter that project_id ||=
> build.try(:project_id)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/912/files/342ea153614715c3f9867aee6ef6c6f63bf8e11c#r61658036>
>